app/components/tailwinds/form/builder.rb

```diff
@@ -29,11 +41,18 @@ def submit(action, **options, &)
 
       private
 
+      def input(method_name)
+        unbound_method = self.class.superclass.instance_method(method_name)
```

Comment:
why you need variable here?
```suggestion
        self.class.superclass.instance_method(method_name).bind(self)
```

Reply:
The `input` method looks strange for usual Ruby application cuz it's some kind of functional programming. So, I decided to make more readable and easy-to-understand :slightly_smiling_face: